File: level 08/classwork/classwork.py

a = 10
b = 20

print(a < b)  # True
print(a > b)  # False
print(a == b)  # False

x = 5
y = 15

print(x < 10 and y > 10)  # True
print(x > 10 or y > 10)  # True

c = 30
d = 40

result = (c < d and c > 20) or c == 30
print(result)  # True

result = (a > b) or (x < 10 and y < 20)
print(result)  # True

File: level 093/classwork/classwork.js

const add = (a, b) => a + b;

const greet = () => console.log("Hello!");

const multiply = (a, b) => {
  const result = a * b;
  return result;
};



const name1 = "Gela";
const greeting = `Hello, ${name1}!`;

const a = 5, b = 10;
const sumText = `Sum is ${a + b}`;



function greet(name2 = "Guest") {
    return `Hello, ${name2}`;
  }

  function sum(a = 0, b = 0) {
    return a + b;
  }

  function createProfile(user = { name: "Anonymous", age: 0 }) {
    return user;
  }
  



  //  Rest
function total(...numbers) {
    return numbers.reduce((sum, n) => sum + n, 0);
  }
  
  //  Spread
  const arr1 = [1, 2, 3];
  const arr2 = [...arr1];
  
  //  Spread
  const user = { name: "Ana" };
  const fullUser = { ...user, age: 25 };
  